net-libs/tox/tox-0.2.9.ebuild

```diff
@@ -0,0 +1,101 @@
+# Copyright 1999-2019 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit cmake-utils systemd user
+
+MY_P="c-toxcore-${PV}"
+DESCRIPTION="Encrypted P2P, messaging, and audio/video calling platform"
+HOMEPAGE="https://tox.chat"
+SRC_URI="https://github.com/TokTok/c-toxcore/archive/v${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0/0.2"
+KEYWORDS="~amd64 ~arm ~x86"
+IUSE="+av daemon ipv6 no-log log-trace log-debug log-info log-warn +log-error static-libs test"
+
+REQUIRED_USE="^^ ( no-log log-trace log-debug log-info log-warn log-error )"
```

Comment:
`log-error` is default enabled, yet it is actually nowhere used except in `REQUIRED_USE`.
On the other hand `no-log` exists, when its meaning would be easily fulfilled by none of `log-lvl` being enabled.
Suggestion: keep `+log-error`, drop `no-log`, flip this condition to `??` and change logic in `src_configure` accordingly.